src/trunk.c

```diff
@@ -8082,11 +8082,11 @@ trunk_print_branches_and_bundles(platform_log_handle *log_handle,
       {
          trunk_subbundle *sb = trunk_get_subbundle(spl, node, sb_no);
          // Generate marker line if curr branch is a sub-bundle's start branch
+         platform_assert(sb->state != SB_STATE_UNINIT);
```

Comment:
Because `trunk_subbundle_filter_count` doesn't change the sb->state at all, the checking would be done right after trunk_get_subbundle.